site/content/docs/cli/cli-vs-terraform-pulumi.md

```diff
@@ -0,0 +1,61 @@
+---
+title: CLI vs. Terraform & Pulumi
+weight: 6
+menu:
+  docs:
+    parent: "CLI"
+---
+
+Creating, debugging and managing E2E and synthetics monitoring at scale is best done "as code". Currently, Checkly supports
+three tools you can use for your Monitoring-as-Code workflow:
+
+1. The Checkly CLI.
+2. Terraform (through our Terraform provider)
+3. Pulumi (through our Pulumi provider)
+
+In most cases, the choice depends on what your goals are and how your organization is set up.
+
+We believe the **Checkly CLI delivers the best-of-breed workflow**. However, there are always trade-offs. Let's list some 
+pros and cons based on our own experience and user feedback.
+
+## Terraform vs. Checkly CLI
```

Comment:
These sections are no real comparisons. Should we just have sections with pros and cons?

## Checkly CLI - Pro / Cons
## TF - Pro / Cons
## Pulumi - Pro / Cons

Reply:
I like it more like this. I think the are comparisons in the form of pros and cons list.

Reply:
Fair enough. :) Still think though, that this page isn't explaining when / where our CLI is better and shines. Maybe more intro and CLI explanation, in the beginning, would do the job.